1-exercises/F-for-of-loop/exercise.js

```diff
@@ -4,13 +4,19 @@
 
 // TODO Use a for-of loop to output each of the tube stations below.
 let tubeStations = [
-    "Aldgate",
-    "Baker Street",
-    "Picadilly Circus",
-    "Oxford Street",
-    "Tottenham Court Road"
+  "Aldgate",
+  "Baker Street",
+  "Picadilly Circus",
+  "Oxford Street",
+  "Tottenham Court Road",
 ];
 
+for (let tubeStation of tubeStations) {
+  console.log(tubeStation);
+}
 
 // TODO Use a for-of loop to capitalise and output each letter in the string seperately.
 let str = "codeyourfuture";
+for (let character of str) {
+  console.log(character);
+}
```

Comment:
Well done Yigit 😊
You have execute the resault nicely.
But there is one more step you need to take which is your output should be capitialised :)

Reply:
Thanks Mo, edited now 😊